pkg/csi/service/wcp/controller.go

```diff
@@ -351,7 +351,7 @@ func (c *controller) ReloadConfiguration(reconnectToVCFromNewConfig bool) error
 		c.manager.VolumeManager.ResetManager(ctx, vcenter)
 		c.manager.VcenterConfig = newVCConfig
 		c.manager.VolumeManager = cnsvolume.GetManager(ctx, vcenter, operationStore,
-			commonco.ContainerOrchestratorUtility.IsFSSEnabled(ctx, common.CSIVolumeManagerIdempotency))
+			commonco.ContainerOrchestratorUtility.IsFSSEnabled(ctx, common.CSIVolumeManagerIdempotency), false)
```

Comment:
Just like how `CSIVolumeManagerIdempotency` is being done.

Reply:
for now, we are making very minimal changes in the WCP logic so supplied as false directly.